catalogs/catalog-hadoop/build.gradle.kts

```diff
@@ -79,6 +79,7 @@ dependencies {
   testImplementation(libs.mockito.core)
   testImplementation(libs.mockito.inline)
   testImplementation(libs.mysql.driver)
+  testImplementation(libs.postgresql.driver)
```

Comment:
Do we need to install pg driver manually when we launch Gravitino?

Reply:
I think we need to update the doc accordingly, to let user understand how to use PG backend.

Reply:
> Do we need to install pg driver manually when we launch Gravitino?

Yes, it's similar to MySQL backend and we need to download the corresponding JDBC driver to the classpath.

Reply:
> I think we need to update the doc accordingly, to let user understand how to use PG backend.

I will use another PR to refine the document about it.